csv_detective/cli.py

```diff
@@ -20,25 +20,40 @@ def run():
         dest='num_rows',
         type=int,
         nargs='?',
-        help='Number of rows to use for detection'
+        help='Number of rows to use for detection (default 500)'
     )
     explorer.add_argument(
-        '-t',
-        '--select_tests',
-        dest='city',
+        '-s',
+        '--sep',
+        dest='sep',
         type=str,
-        nargs='*',
-        help='List of tests to be performed (use "" if you want to use the dash option to remove tests)'
+        nargs='?',
+        help='Columns separator'
+    )
+    explorer.add_argument(
+        '--save',
+        dest='save_results',
+        type=int,
+        nargs='?',
+        help="Whether to save the resulting analysis to json (1 = save, 0 = don't)"
+    )
+    explorer.add_argument(
+        '-v',
+        '--verbose',
+        dest='verbose',
+        type=int,
+        nargs='?',
+        help='Verbose (0 = quiet, 1 = details)'
     )
 
     opts = explorer.parse_args()
 
-    num_rows = opts.num_rows or 50
     inspection_results = routine(
-        opts.file_path,
-        num_rows=num_rows,
-        user_input_tests='ALL',
-        output_mode='ALL'
+        csv_file_path=opts.file_path,
+        num_rows=opts.num_rows or 500,
```

Comment:
nit: related to the previous suggestion
```suggestion
        num_rows=opts.num_rows,
```